roxctl/sensor/generate/generate.go

```diff
@@ -89,6 +91,10 @@ func (s *sensorGenerateCommand) Construct(cmd *cobra.Command) error {
 		s.env.Logger().WarnfLn(warningDeprecatedAdmControllerEnableSet)
 	}
 
+	if cmd.PersistentFlags().Lookup(slimCollector).Changed {
+		s.env.Logger().WarnfLn(warningSlimCollectorModeSet)
+	}
+
```

Comment:
is this equal to (excluding the flag will be hidden)
```
	utils.Must(c.PersistentFlags().MarkDeprecated(slimCollector, warningSlimCollectorModeSet)
```

Reply:
Done